### PR TITLE
Update shadowsocks-libev-debian.sh

### PR DESCRIPTION
**Pull request by LiveChief**
Updated a little bit of the script.

No Script change more of a proper grammer change type of way.

### CODE REVIEW

File: shadowsocks-libev-debian.sh [hunk 1]
```diff
@@ -279,7 +279,7 @@ pre_install(){
             break
         fi
     fi
-    echo -e "[${red}Error${plain}] Please enter a correct number [1-65535]"
+    echo -e "[${red}Error${plain}] Please enter a port number (Recommended 80 or 443) [1-65535]"
```
Comment (hybtoy) [inline shadowsocks-libev-debian.sh]: > (Recommended 80 or 443)
Definitely not recommended. With 443 port, IP will be banned in 30 minutes or less.

Comment (LiveChief) [reply shadowsocks-libev-debian.sh]: which ones do u recommend?

Comment (hybtoy) [reply shadowsocks-libev-debian.sh]: above 10000

Comment (hybtoy) [reply shadowsocks-libev-debian.sh]: > which ones do u recommend?

above 10000

**Comment by LiveChief** [comment]
1-65535
Choose a coupe that u like and u would recommend, cause I feel like people just leave it as default and it's not hard to find..

File: shadowsocks-libev-debian.sh (hunk 1 above)
Comment (LiveChief) [reply shadowsocks-libev-debian.sh]: 1-65535
Choose a coupe that u like and u would recommend, cause I feel like people just leave it as default and it's not hard to find..

**Pull request closed without merging**